server/user/tests/test_pk.py

```diff
@@ -42,6 +56,7 @@ def setUp(self):
     @parameterized.expand(
         [
             ("user2", status.HTTP_403_FORBIDDEN),
+            ("user3", status.HTTP_200_OK),
             (None, status.HTTP_401_UNAUTHORIZED),
         ]
     )
```

Comment:
Help me plz to read the test / functionality properly. As far as I understand from the code, the following happens:
1. we log in under the `user3`
2. we request some `user1` profile info (`url` is set up with `user1_id`)
3. we expect that `user3` have access to `user1` profile info
Right?

And the test works as follows:
1. `user3` (benchmark owner in reality?) creates an mlcube (data prep mlcube?)
2. `user1` creates dataset that includes aforementioned  data prep mlcube
3. So, benchmark owner has an access to profiles of users - connected datasets' owners

If yes, let me ask a dummy question. May dataset be connected to any other mlcubes different from dataprep, say, to some model mlcube? In this case would (and should) model mlcube owner have an access to dataset's owner profile?

Reply:
Correct, although in this case we don't assume `user3` to be a benchmark owner necessarily, but a data prep mlcube owner. There's probably going to be a good intersection between those two roles, but not necessarily always.

Regarding your question with model mlcube, it's a good one, and one that will require discussions with the team. The reason this change had to happen here is because we need the users emails to be able to display that information in the dashboard and know who to contact if something is failing

Reply:
Hm. If there may be a difference between benchmark owner and data prep cube owner, shouldn't we give an access to the BO instead of data prep owner?

Reply:
@VukW that's true; for the experiment owner not the benchmark owner since in RANO there is no benchmark. but since until this moment we don't have training experiment logic, we had only prep cube so the current permissions are temporary until all hospitals associate with the training experiment that we will create soon